packages/picasso/src/Container/styles.ts

```diff
@@ -55,6 +55,14 @@ const paddings = spacingVariants.reduce((acc, variant) => {
   return acc
 }, Object.create(null))
 
+const gaps = spacingVariants.reduce((acc, variant) => {
+  acc[`${variant}Gap`] = {
+    gap: spacingToRem(variant as SpacingType)
+  }
+
+  return acc
+}, Object.create(null))
```

Comment:
wdyt guys if we join this loop with the previous one:
```suggestion
const paddings = spacingVariants.reduce((acc, variant) => {
  acc[`${variant}Padding`] = acc[`${variant}Gap`] = {
    padding: spacingToRem(variant as SpacingType)
  }
  return acc
}, Object.create(null))
```

Reply:
I don't have a strong opinion against it, but I don't like it much, suddenly `paddings` would have to be renamed to `paddingsAndGaps` and we would mix responsibilities.